Dockerfile

```diff
@@ -1,4 +1,17 @@
-FROM horstbaerbel/ccpp-cmake-build-and-test:1.0
+FROM ubuntu:rolling
+
+WORKDIR /
+
+ENV TZ=Europe/Berlin
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
I don't understand this. Why do we need `TZ=Europe/Berlin`? Is this the minimal amount of code to do the job?

Reply:
That's literally a copy-paste from the upstream repo.

I suspect, we need to tell ubuntu the machine is in. It doesn't matter too much I think.

Some application installs do require timezone to be setup... I think as we are doing a dist-upgrade and install, they might pull packages that require it.

I'm not sure.